.github/workflows/release.yml

```diff
@@ -6,9 +6,16 @@ on:
   push:
     tags:
       - "*" # triggers only if push new tag version, like `0.8.4`.
+  # Run daily as a dry-run/test.
+  schedule:
+    - cron: '0 1 * * *'
 
 permissions: read-all
 
+env:
+  GH_TOKEN: ${{ secrets.E2E_GO_TOKEN }}
```

Comment:
make sure we have this token defined in this repo.

Reply:
aaah! actually i do not have access to add my PAT to the repository. maybe because i'm not an org admin

Reply:
Can the default GITHUB_TOKEN create issues? I think it can... Have you checked?

Reply:
I think it will! it has issue read/write perms. will remove this one https://docs.github.com/en/actions/security-guides/automatic-token-authentication#permissions-for-the-github_token